src/EditorFeatures/Core.Wpf/Lightup/ISmartRenameSessionWrapper.cs

```diff
@@ -17,83 +19,94 @@ namespace Microsoft.CodeAnalysis.EditorFeatures.Lightup;
 internal readonly struct ISmartRenameSessionWrapper : INotifyPropertyChanged, IDisposable
 {
     internal const string WrappedTypeName = "Microsoft.VisualStudio.Text.Editor.SmartRename.ISmartRenameSession";
-    private static readonly Type s_wrappedType;
+    private static readonly Type? s_wrappedType;
 
-    private static readonly Func<object, bool> s_isAvailableAccessor;
-    private static readonly Func<object, bool> s_hasSuggestionsAccessor;
-    private static readonly Func<object, bool> s_isInProgressAccessor;
-    private static readonly Func<object, string> s_statusMessageAccessor;
-    private static readonly Func<object, bool> s_statusMessageVisibilityAccessor;
-    private static readonly Func<object, IReadOnlyList<string>> s_suggestedNamesAccessor;
+    private static readonly Func<object, bool>? s_isAvailableAccessor;
+    private static readonly Func<object, bool>? s_hasSuggestionsAccessor;
+    private static readonly Func<object, bool>? s_isInProgressAccessor;
+    private static readonly Func<object, string>? s_statusMessageAccessor;
+    private static readonly Func<object, bool>? s_statusMessageVisibilityAccessor;
+    private static readonly Func<object, IReadOnlyList<string>>? s_suggestedNamesAccessor;
```

Comment:
None of these should be nullable. If the member cannot be bound, an implementation returning a default value should be used instead.